code/modules/spells/spell_types/wizard.dm

```diff
@@ -229,6 +231,18 @@
 	charge_max = 5000
 	summon_amt = 2
 
+/obj/effect/proc_holder/spell/aoe_turf/conjure/creature/bee
+	name = "Lesser summon bees"
+	desc = "This spell magically kicks a transdimensional beehive, instantly summoning a swarm of bees to your location."
+	charge_max = 1000
+	clothes_req = TRUE
+	invocation = "NOT THE BEES"
+	summon_amt = 9
+	action_icon_state = "bee"
+
+	summon_type = /mob/living/simple_animal/hostile/poison/bees/toxin
```

Comment:
give these bees a faction that matches that of the wizard that summons them, or else this is just going to be an elaborate suicide spell (unless you're an android, you're a podperson, or you're wearing a gem-encrusted hardsuit)

Reply:
Alternatively, the wizard gains that faction upon purchase of the spell.

Reply:
Bees simply just have the "hostile" faction, which means if I give the wizard that faction, every hostile mob will ignore them

Reply:
add a bee faction and make bees (and podpeople) use that instead of the weird snowflakey reaction bees current have towards podpeople

Reply:
> add a bee faction and make bees (and podpeople) use that instead of the weird snowflakey reaction bees current have towards podpeople

just use the plants faction